docs/api/ctx.md

```diff
@@ -2202,3 +2203,24 @@ app.Get("/", func(c fiber.Ctx) error {
   // </Fiber>
 })
 ```
+
+## Convert 
+Converts a string value to a specified type, handling errors and optional default values.
+You can combine this function with other methods and functions.
+
+```go title="Signature"
+func Convert[T any](value string, convertor func(string) (T, error), defaultValue ...T) (*T, error)
+```
+
+```go title="Example"
+  fiber.Convert("true", strconv.ParseBool) // true, nil
```

Comment:
This doc section should probably explain why someone would use fiber.Convert instead of the conversion function directly.

Reply:
@coderabbitai can you suggest a good improvement for these documentation and also consider this comment


> This doc section should probably explain why someone would use fiber.Convert instead of the conversion function directly.

Reply:
@dozheiny can you use it to improve the documentation so that we can merge directly afterwards ?

Reply:
> @dozheiny can you use it to improve the documentation so that we can merge directly afterwards ?

sure